src/main/java/org/asciidoctor/asciidoclet/AsciidoctorRenderer.java

```diff
@@ -127,7 +127,28 @@ private void renderTag(Tag tag, StringBuilder buffer) {
         //print out directly
         buffer.append(tag.name());
         buffer.append(" ");
-        buffer.append(render(tag.text(), true));
+        if ("@param".equals(tag.name())) {
+            // Special handling for @param <T> tags
+            // See http://docs.oracle.com/javase/1.5.0/docs/tooldocs/windows/javadoc.html#@param
+            String parameterName = getParameterName(tag.text());
+            buffer.append(parameterName).append(' ');
+            buffer.append(render(tag.text().replace(parameterName, ""), true));
```

Comment:
You know the length of the parameter name (plus the space), so I would use substring here instead of replace.